docs/API/InstrumentFormat.md

```diff
@@ -237,11 +287,12 @@ as follows:
 
 `Name`: Required. Follows `PageElement.Name` rules.
 
-`Description`: Required. Follows `PageElement.Name` rules.
-
-`Options.MinDate`: The minimum date that can be chosen by the user. Format is YYYY-MM-DD
+`Description`: Required. Follows `PageElement.Description` rules.
 
-`Options.MaxDate`: The maximum date that can be chosen by the user. Format is YYYY-MM-DD
+```
+Currently no min/max date is implemented. Date elements are unrestricted.
+ZV
```

Comment:
They're currently supported by instruments created from the instrument builder and need to continue to be supported. Same with regex. (I think? I'm not 100% sure if regex validation was ever implemented, but if not, it's an implementation bug, not a spec change.)

Reply:
Regex was only supported (dodgily) through Xin or form rules (i forget), but not via Instrument builder.